ocs_ci/helpers/ocs_metrics_exporter_helpers.py

```
@@ -0,0 +1,274 @@
+
+def assert_prometheus_exposition_text(text):
+    """
+    Assert the payload looks like Prometheus text exposition (not HTML/JSON error page).
+
+    Args:
+        text (str): body from /metrics
+
+    Raises:
+        AssertionError: if body does not match minimal Prometheus text format heuristics.
+    """
+    assert text and text.strip(), "metrics endpoint returned an empty body"
```

Ruff (0.15.15)

[warning] Assertion should be broken down into multiple parts

(PT018)